contracts/deploy/036_multiple_rewards_strategies.js

```diff
@@ -251,59 +253,63 @@ module.exports = deploymentWithProposal(
           contract: cHarvester,
           // tokenAddress, allowedSlippageBps, harvestRewardBps, uniswapV2CompatibleAddr, liquidationLimit, doSwapRewardToken
           signature:
-            "setRewardTokenConfig(address,uint16,uint16,address,uint256,bool)",
+            "setRewardTokenConfig(address,uint16,uint16,address,uint256,bool,uint24)",
```

Comment:
as in the 037 migration file, I think this file doesn't need any changes, since it has already been executed on the mainnet

Reply:
You will probably need to create a fresh new deploy file (038...) in order to be able to upgrade the harvester.